src/Modules/UngradedModules.jl

```diff
@@ -1112,11 +1247,41 @@ end
 @doc Markdown.doc"""
     SubQuo(A::MatElem{R}, B::MatElem{R}) where {R}
 
-Let $A$ be an $n_1 \times m$-matrix, $B$ an $n_2 \times m$-matrix over the ring $R$. 
-Then return the subquotient $(\im(A) + \im(B))/ \im(B)$. 
-Note: The embedding free module of the subquotient is constructed by the function and therefore
-not compatible with free modules $R^m$ that are defined by the user or other functions. 
-If you need compatibility use `SubQuo(F::FreeMod{R}, A::MatElem{R}, B::MatElem{R})`.
+Given matrices `A` and `B` with entries in a ring `R` 
+representing maps of free $R$-modules with the same codomain,
+return the subquotient $(\text{im } A + \text{im }  B)/\text{im }  B.$ 
+
+NOTE: The ambient free module of the subquotient is constructed by the function and therefore
```

Comment:
Consider using [the admonition syntax](https://docs.julialang.org/en/v1/stdlib/Markdown/#Admonitions) here.